packages/xrpc/src/util.ts

```diff
@@ -144,7 +145,7 @@ export function httpResponseBodyParse(
     if (mimeType.includes('application/json') && data?.byteLength) {
       try {
         const str = new TextDecoder().decode(data)
-        return JSON.parse(str)
+        return jsonStringToIpld(str)
```

Comment:
Might be good to get an early read on whether `@atproto/api` will still work on clients where we need it to.  Not sure how some deps in `@atproto/common` including `multiformats/cid` will play on the web, for example.